cmake/3rdParty/Platform/Linux/BuiltInPackages_linux.cmake

```diff
@@ -44,3 +45,6 @@ ly_associate_package(PACKAGE_NAME qt-5.15.2-rev4-linux
 ly_associate_package(PACKAGE_NAME libsamplerate-0.2.1-rev2-linux                    TARGETS libsamplerate               PACKAGE_HASH 41643c31bc6b7d037f895f89d8d8d6369e906b92eff42b0fe05ee6a100f06261)
 ly_associate_package(PACKAGE_NAME OpenSSL-1.1.1b-rev2-linux                         TARGETS OpenSSL                     PACKAGE_HASH b779426d1e9c5ddf71160d5ae2e639c3b956e0fb5e9fcaf9ce97c4526024e3bc)
 ly_associate_package(PACKAGE_NAME DirectXShaderCompilerDxc-1.6.2104-o3de-rev2-linux TARGETS DirectXShaderCompilerDxc    PACKAGE_HASH 235606f98512c076a1ba84a8402ad24ac21945998abcea264e8e204678efc0ba)
+
+# todo: need spirv linux version
+ly_associate_package(PACKAGE_NAME SPIRVCross-2021.04.29-rev1-mac                    TARGETS SPIRVCross                  PACKAGE_HASH 78c6376ed2fd195b9b1f5fb2b56e5267a32c3aa21fb399e905308de470eb4515)
```

Comment:
Using Mac version and manually replacing it with the Linux version after build.

Reply:
I believe @spham-amzn was looking to getting getting the azslc and SPRIVCross packages working on linux

Reply:
This package was completed, it just needs to be promoted to production. Will update once its there

Reply:
Ideally I would merge in shader changes once the azslc binaries are uploaded but its fine if we do it a bit early.

Reply:
This specific package is ready for review ( https://github.com/o3de/o3de/pull/2233 )